packages/webview_flutter/example/lib/main.dart

```diff
@@ -55,26 +55,60 @@ class WebViewExample extends StatelessWidget {
   }
 }
 
+enum MenuOptions {
+  evaluateJavascript,
+  toast,
+}
+
 class SampleMenu extends StatelessWidget {
-  const SampleMenu();
+  SampleMenu(this.controller);
+  final Future<WebViewController> controller;
 
   @override
   Widget build(BuildContext context) {
-    return PopupMenuButton<String>(
-      onSelected: (String value) {
-        Scaffold.of(context)
-            .showSnackBar(SnackBar(content: Text('You selected: $value')));
+    return FutureBuilder<WebViewController>(
+      future: controller,
+      builder:
+          (BuildContext context, AsyncSnapshot<WebViewController> controller) {
+        return PopupMenuButton<MenuOptions>(
+          onSelected: (MenuOptions value) {
+            switch (value) {
+              case MenuOptions.evaluateJavascript:
+                _onEvaluateJavascript(controller.data, context);
+                break;
+              case MenuOptions.toast:
+                Scaffold.of(context).showSnackBar(
+                  SnackBar(
+                    content: Text('You selected: $value'),
+                  ),
+                );
+                break;
+            }
+          },
+          itemBuilder: (BuildContext context) => <PopupMenuItem<MenuOptions>>[
+                PopupMenuItem<MenuOptions>(
+                  value: MenuOptions.evaluateJavascript,
+                  child: const Text('Evaluate JavaScript (change bg)'),
```

Comment:
It might be unclear what 'bg' means, I'd either make this "change background color", or completely drop that part.